CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+CHANGES IN VERSION 1.11.1 (from 1.11.0)
+===================================
+
+Appium 1.11.1 is a patch release
+
+#### General
+* Upgrade Appium You.I Engine Driver to 1.1.2
+* Fix protocol translation bug (W3C -> MJSONWP) in `/actions` [#302](https://github.com/appium/appium-base-driver/pull/302)
+* Fix protocol conversion for `setValue` [#297](https://github.com/appium/appium-base-driver/pull/297)
+
+#### Android (UiAutomator2 only)
```

Comment:
Is it better to address about improving checking instrumentation process https://github.com/appium/appium-uiautomator2-driver/pull/284 ?
The change is for https://github.com/appium/appium-uiautomator2-driver/pull/282 which was introduced in 1.11.0

It can fix https://github.com/appium/appium/issues/12167